lib/types/row.js

```diff
@@ -13,17 +37,24 @@ function Row(columns) {
  */
 Row.prototype.get = function (columnName) {
   if (typeof columnName === 'number') {
-    if (this.__columns && this.__columns[columnName]) {
-      columnName = this.__columns[columnName].name;
-    }
-    else {
-      throw new Error('Column not found');
-    }
+    return this._getValueFunc(columnName)();
```

Comment:
:+1: that sounds good to me, didn't occur to me at the time that the other drivers worked that way.